task1_declaration.js

```diff
@@ -0,0 +1,8 @@
+function rectangleArea(width, height) {
```

Comment:
function name should represent the action e.g. `calculateRectangleArea`,  `getRectangleArea` etc

Reply:
same for other implementations